### ISSUE TITLE
Can't tab complete on ZSH without dash args

### ISSUE THREAD

**Issue by vinieich** [rmarquis/pacaur#771]
[tst.txt](https://github.com/rmarquis/pacaur/files/1455875/tst.txt)
###### Version

`pacaur -v`

```
pacaur 4.7.10.34.gb4a4932
```

###### Description

Can't tab complete the "sync, clone, check, etc..." arguments on ZSH without any zsh-framework.
Tried to solve it with zsh completion file but no luck. What I've learned was that it won't accept any argument without dash. If I put any argument like "--foobar" on "sync" place it will work.
Hope it's clear, english is not my primary language.
Thanks in advance

###### Output

`pacaur <TAB>`

```
_arguments:comparguments:319: invalid argument: {sync}[Clone, build and install target(s)]

```

###### Debug output

`bash -x pacaur <your command>`

```
Paste result of above command here
```
`
on attachment
`

**Comment by rmarquis** [comment]
Last time I checked, this was working on my machine. I might have introduced some issue in the few last git commits. I'll have a look when I get access to my Linux machine again (in a few weeks).

**Comment by vinieich** [comment]
I've spent all day today trying to figure out how to make this thing work as I'm not familiar with programming nor scripting. But after a long research on zsh docs I think I got it.
I created a fork with the solution. Is this the correct way?
#772

**Comment by rmarquis** [comment]
Okay, I'll review the changes when I'm able to.

**Comment by rmarquis** [comment]
Fixed through #772.
